### ISSUE TITLE
How to enable password strength on standalone registration component?

### ISSUE THREAD

**Issue by crunck78** [AnthonyNahas/ngx-auth-firebaseui#609]
### Bug Report or Feature Request (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [x ] feature request
```

### Desired functionality

Hi Anthony.  Thanks again for this great lib. I use the registration and login components as standalone. Seems like the password strength and length only works for the auth component. Will there be one for the standalone registration component?

**Comment by geromegrignon** [comment]
@AnthonyNahas this issue can be closed thanks to #610

**Comment by AnthonyNahas** [comment]
I've just release v5.1.0 

thank you all and cheers
